Source/Scene/SceneTransitioner.js

```diff
@@ -292,7 +292,7 @@ define([
 
     function destroyMorphHandler(transitioner) {
         var tweens = transitioner._currentTweens;
-        for ( var i = 0; i < tweens.length; ++i) {
+        for (var i = 0; i < tweens.length; ++i) {
```

Comment:
Sorry to nitpick: All the whitespace changes are unnecessary! While it doesn't really matter if there is whitespace or not, it makes the real changes of this PR hard to see.